SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -4,6 +4,8 @@
 
 #include "Interop/Connection/OutgoingMessages.h"
 #include "Interop/Connection/SpatialOSWorkerInterface.h"
+// TODO remove
```

Comment:
Is it time? :)

Reply:
Hmm you talking about removing the event tracer from WorkerConnection or the header?

Reply:
Yeah, or removing the comment if its too much effort (I think we looked at this before and involved a lot of shuffling)